### PR TITLE
Add a Skill-Page

### PR DESCRIPTION
**Pull request by sweisz**
- Skill-Page with Buttons, Logo and Headers components

### CODE REVIEW

File: components/button/Button.module.css [hunk 1]
```diff
@@ -1,11 +1,13 @@
 .btnSkills {
-  font-family: var(--primary-font-family);
   background-color: #3c381d5e;
   border: none;
   border-radius: 6px;
   color: white;
+  font-family: var(--primary-font-family);
```
Comment (mewiemers) [inline components/button/Button.module.css:6]: is this a font in your global? another name would be better

**Review by mewiemers — COMMENTED** [review]
GJ!🎸

**Review by SilviaBarcellini — APPROVED** [review]
Nice 👍

File: components/button/Button.module.css (hunk 1 above)
Comment (sweisz) [reply components/button/Button.module.css:6]: tx, it is in global, but i have 3 fonts and want to separate it. Naming is a topic ;)

**Pull request merged**